mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/routes/v1/ArenaRoutes.kt

```diff
@@ -129,6 +126,31 @@ fun Route.arenaRoutes() {
                     call.respond(HttpStatusCode.InternalServerError, "Kunne ikke slette tiltakshistorikk")
                 }
         }
+
+        put("deltaker") {
+            val deltaker = call.receive<DeltakerDbo>()
+
+            arenaService.upsertDeltaker(deltaker)
+                .onRight { call.respond(it) }
+                .onLeft {
+                    logError(logger, it.error)
```

Comment:
Kan det potensielt være tilfeller der `it.error` kan logge persondata nå eller på et senere tidspunkt? Er det tryggere å logge med secureLog fra starten av?

Reply:
Fra starten av er nok litt sent da dette har pågått en stund :smile: 

Slike psqlexceptions skal ikke inneholde selve dataen vi forsøker å inserte, men enig i at det er tryggere å logge stacktraces til securelog når vi vet at brukerdata kan være involvert :+1: 

Fikser opp i loggingen i separat PR